tests/conftest.py

```diff
@@ -39,6 +40,21 @@ def hosts(tmpdir_factory):
     }
 
 
+@pytest.fixture(scope='function', autouse=False)
+def author(request):
```

Comment:
I struggled for a brief time thinking this should modify the return values of the actual `author()` functions. So maybe just add a comment that it calculates expected author values:

```suggestion
def author(request):
    r"""Return expected author values."""
```